server/db/migrate/h2deltas/1909002_remove_artifact_plans_for_completed_jobs.sql

```diff
@@ -0,0 +1,30 @@
+--
+-- Copyright 2019 ThoughtWorks, Inc.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--    http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+
+DELETE FROM artifactPlans
+WHERE artifactPlans.id IN (
+  SELECT artifactPlans.id
+  FROM artifactPlans
+  INNER JOIN builds
+    ON builds.id = artifactPlans.buildId
+  WHERE
+      artifactPlans.artifactType = 'file'
```

Comment:
Why don't we get rid of artifactPlans.artifactType = 'unit' too?

Reply:
See commit message at 23eb465f0b366378c526c70642c3d0748ce47e2e. I don't think that's relevant anymore. So I suppose this part of the `WHERE` clause can be dropped.

Reply:
Exactly, didn't shine go away?

Reply:
It did, so did this `WHERE` clause.

Reply:
How long does this query take to get rid of records now?

Reply:
```
SELECT COUNT(*) from artifactPlans;
COUNT(*)  
762480
(1 row, 0 ms)

DELETE FROM artifactPlans
WHERE artifactPlans.id IN (
  SELECT artifactPlans.id
  FROM artifactPlans
  INNER JOIN builds
    ON builds.id = artifactPlans.buildId
  WHERE
      builds.state = 'Completed'
);
Update count: 582665
(61587 ms)

SELECT COUNT(*) from artifactPlans;
COUNT(*)  
179815
(1 row, 0 ms)
```